packages/pipeline/src/scripts/scrape_data.ts

```diff
@@ -98,7 +106,29 @@ export const scrapeDataScripts = {
                 }
               });
         });
-    }
+    },
+    getHistoricalPriceSeries(fromSymbol: string,
+                             toSymbol: string,
+                             fromTimestamp: number,
+                             toTimestamp: number): any {
+        return new Promise((resolve, reject) => {
+            const limit = Math.min(
+                Math.round((toTimestamp - fromTimestamp) / (86400 * 1000)), 2000);
```

Comment:
Let's not leave magic constants in the codebase. Please move these into clearly named constants.

Reply:
These are dynamically defined, but cleaned up for descriptiveness!